tests/test_tracking_grid/results_true.dat

```diff
@@ -0,0 +1,27 @@
+Diagonal track...
+Number of segments = 3
```

Comment:
"It is certainly a tradeoff between transparency and data footprint. I would just argue that in order to debug one of these tracking tests, a developer would have to understand how the test works by looking at "test_tracking_*.py" anyway, which would lessen the need to have all of this redundant metadata.